src/functions/template-tags/date.php

```diff
@@ -14,117 +14,48 @@
-	 * @param string $timezone    Timezone in which to present the date/time (or default behaviour if not set)
-	 *
-	 * @return string|null Time
-	 */
-	function tribe_get_start_time( $event = null, $dateFormat = '', $timezone = null ) {
-		if ( is_null( $event ) ) {
-			global $post;
-			$event = $post;
-		}
-
-		if ( is_numeric( $event ) ) {
-			$event = get_post( $event );
-		}
-
-		if ( ! is_object( $event ) ) {
-			return;
-		}
-
-		if ( tribe_event_is_all_day( $event ) ) {
-			return;
-		}
-
-		$start_date = Tribe__Events__Timezones::event_start_timestamp( $event->ID, $timezone );
-
-		if ( '' == $dateFormat ) {
-			$dateFormat = tribe_get_time_format();
-		}
-
-		return tribe_format_date( $start_date, false, $dateFormat );
-	}
-}
-
-if ( ! function_exists( 'tribe_get_end_time' ) ) {
-	/**
-	 * End Time
-	 *
-	 * Returns the event end time
-	 *
-	 * @category Events
-	 * @see http://php.net/manual/en/function.date.php
-	 *
-	 * @param int    $event       (optional)
-	 * @param string $dateFormat  Allows date and time formating using standard php syntax
-	 * @param string $timezone    Timezone in which to present the date/time (or default behaviour if not set)
-	 *
-	 * @return string|null Time
-	 */
-	function tribe_get_end_time( $event = null, $dateFormat = '', $timezone = null ) {
-		if ( is_null( $event ) ) {
-			global $post;
-			$event = $post;
-		}
-
-		if ( is_numeric( $event ) ) {
-			$event = get_post( $event );
-		}
-
-		if ( ! is_object( $event ) ) {
-			return;
-		}
-
-		if ( tribe_event_is_all_day( $event ) ) {
-			return;
-		}
-
-		$end_date = Tribe__Events__Timezones::event_end_timestamp( $event->ID, $timezone );
-
-		if ( '' == $dateFormat ) {
-			$dateFormat = tribe_get_time_format();
-		}
-
-		return tribe_format_date( $end_date, false, $dateFormat );
-	}
-}
-
 if ( ! function_exists( 'tribe_get_display_end_date' ) ) {
 	/**
 	 * End Date formatted for display
 	 *
 	 * Returns the event end date that observes the end of day cutoff
 	 *
 	 * @category Events
-	 * @see http://php.net/manual/en/function.date.php
+	 * @see      http://php.net/manual/en/function.date.php
 	 *
-	 * @param int    $event        (optional)
-	 * @param bool   $display_time If true shows date and time, if false only shows date
-	 * @param string $date_format  Allows date and time formating using standard php syntax
-	 * @param string $timezone     Timezone in which to present the date/time (or default behaviour if not set)
+	 * @param int|WP_Post $event        The event (optional).
+	 * @param bool        $display_time If true shows date and time, if false only shows date.
+	 * @param string      $date_format  Allows date and time formatting using standard php syntax.
+	 * @param string      $timezone     Timezone in which to present the date/time (or default behaviour if not set).
 	 *
 	 * @return string|null Date
 	 */
 	function tribe_get_display_end_date( $event = null, $display_time = true, $date_format = '', $timezone = null ) {
```

Comment:
By the looks of this PR we are deprecating one method already? Right?
I was suggesting that we "soft-deprecated" by pointing both the new one.